src/appleseed-max-impl/appleseedrenderer/renderersettings.cpp

```diff
@@ -166,15 +166,14 @@ const char* get_shader_override_type(const int shader_override_type)
       case 12: return "original_shading_normal";
       case 13: return "primitives";
       case 14: return "ray_spread";
-      case 15: return "regions";
-      case 16: return "screen_space_wireframe";
-      case 17: return "shading_normal";
-      case 18: return "sides";
-      case 19: return "tangent";
-      case 20: return "uv";
-      case 21: return "world_space_position";
-      case 22: return "world_space_velocity";
-      case 23: return "world_space_wireframe";
+      case 15: return "screen_space_wireframe";
```

Comment:
This breaks compatibility with existing scenes. Admittedly, it's not a very serious breakage, but the solution is very simple: change

```cpp
case 15: return "regions";

```

to

```cpp
case 15: return "coverage";    // regions are no longer supported
```

What do you think?

Reply:
I had no problems encountered with the existing scenes I used to test. Maybe if someone had saved the scene with the Regions surface shader enabled it would be remapped now to Screen Space Wireframe (case 15).
But then we would need to keep the UI menu entry, otherwise we would have more cases than surface shader override entries?

Reply:
All diagnostic modes >= 15 would be affected actually, not just the Regions mode.

I think you could do _as if_ you had simply removed the Regions mode, but still remap value 15 ("Regions") to value 6 ("Coverage") or maybe better, 0 ("No Override") when loading the project from disk.

That said I doubt we'll ever hit any problem here so let's ignore this altogether :)